lib/dgram.js

```diff
@@ -121,6 +122,12 @@ function Socket(type, listener) {
 
   // If true - UV_UDP_REUSEADDR flag will be set
   this._reuseAddr = options && options.reuseAddr;
+  
+  if (options && options.recvBufferSize)
```

Comment:
@cjihrig I guess extending this PR to add functionality to the interface in JS layer to use `getsockopt` down in libuv might be a good idea (e.g. `getRecvBufferSize()`). Therefore the symbol values holding initial options and can be ignored after they are set against the socket when the `listening` event triggers.